modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/FragmentEditor.es.js

```diff
@@ -136,11 +136,13 @@ class FragmentEditor extends PortletBase {
 	_handleSaveButtonClick(event) {
 		const content = this.getContent();
 		const status = event.delegateTarget.value;
+		
+		console.log('FRAGMENT SAVE LOG');
 
 		if (this.isHtmlValid()) {
 			this._saving = true;
 
-			this.fetch(this.urls.edit, {
+			fetch(this.urls.edit, {
```

Comment:
This code would fail. The second argument of `Liferay.Util.fetch` is not the same as `PortletBase.fetch`. The former is init object, the latter object literal that is to be transferred to `body` option.